docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wjnIQWX8d5UY"
+   },
+   "source": [
+    "# Mitiq paper codeblocks\n",
+    "\n",
+    "Code blocks from the main text of the Mitiq paper [1], updated to [arXiv v2](https://arxiv.org/abs/2009.04417)."
```

Comment:
I would not specify "v2" for future proofing. Just linking to arXiv is sufficient as this shows the latest version.

Reply:
Ok i'll resolve that.